src/symbols/Node.tsx

```diff
@@ -350,7 +352,8 @@ export const Node: FC<NodeProps> = ({
                 stroke={theme.node.subLabel?.stroke}
                 active={isSelected || active || isDraggingCurrent || isActive}
                 color={
-                  isSelected || active || isDraggingCurrent || isActive
+                  (isSelected || active || isDraggingCurrent || isActive) &&
```

Comment:
Any chance you could consolidate these checks into a boolean thats easier to follow above?